treeherder/config/settings_local.example.py

```diff
@@ -1,38 +0,0 @@
-# Switch to using a different bugzilla instance
-BZ_API_URL = "https://bugzilla-dev.allizom.org"
```

Comment:
Should we somehow make the bugzilla instance depend on an environment variable, and use a different value for vagrant?